src/phases/execution/config/MultiDeletes/Secondary.yml

```diff
@@ -0,0 +1,30 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Configuration for MultiDeletes workload. This configuration uses a secondary index to delete.
+
+GlobalDefaults:
+  # The size of the documents inserted and deleted into collections during this test.
+  DocumentSizePaddingBytes: &DocumentSizePaddingBytes {^Parameter: {Name: "DocumentSizePaddingBytes", Default: 0}}
+
+InitializeDatabaseOps:
+- OperationName: RunCommand
+  OperationCommand: {dropDatabase: 1}
+- OperationName: RunCommand
```

Comment:
To double check my understanding:

Are these commands 
(1) part of the same phase
(2) run in parallel, or run serially in the same phase

Reply:
These code is "injected" in place into the RunCommand actor in Phase 1. This actor will run the defined operations in order, serially within Phase 1.